### PR TITLE
feat: migrate odpf to goto

### PR DESCRIPTION
**Pull request by krtkvrm**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -47,13 +47,13 @@ dockers:
...
     image_templates:
-      - "docker.io/odpf/{{.ProjectName}}:latest"
-      - "docker.io/odpf/{{.ProjectName}}:{{ .Version }}"
-      - "docker.io/odpf/{{.ProjectName}}:{{ .Tag }}-amd64"
+      - "docker.io/gotocompany/{{.ProjectName}}:latest"
+      - "docker.io/gotocompany/{{.ProjectName}}:{{ .Version }}"
+      - "docker.io/gotocompany/{{.ProjectName}}:{{ .Tag }}-amd64"
 nfpms:
-  - maintainer: ODPF
+  - maintainer: goto
```
Comment (mabdh) [inline .goreleaser.yml]:
```suggestion
  - maintainer: gotocompany
```

File: README.md [hunk 2]
```diff
@@ -81,20 +81,20 @@ scoop update shield
 We provide ready to use Docker container images. To pull the latest image:
 
 ```
-docker pull odpf/shield:latest
+docker pull goto/shield:latest
```
Comment (mabdh) [inline README.md]:
```suggestion
docker pull gotocompany/shield:latest
```

File: README.md [hunk 3]
```diff
@@ -81,20 +81,20 @@ scoop update shield
...
 ```
 
 To pull a specific version:
 
 ```
-docker pull odpf/shield:v0.3.2
+docker pull goto/shield:v0.3.2
```
Comment (mabdh) [inline README.md]:
```suggestion
docker pull gotocompany/shield:v0.3.2
```

File: README.md [hunk 4]
```diff
@@ -81,20 +81,20 @@ scoop update shield
...
 ```
 
 If you like to have a shell alias that runs the latest version of shield from docker whenever you type `shield`:
 
 ```
-mkdir -p $HOME/.config/odpf
-alias shield="docker run -e HOME=/tmp -v $HOME/.config/odpf:/tmp/.config/odpf --user $(id -u):$(id -g) --rm -it -p 3306:3306/tcp odpf/shield:latest"
+mkdir -p $HOME/.config/goto
```
Comment (mabdh) [inline README.md]:
```suggestion
mkdir -p $HOME/.config/gotocompany
```

File: README.md [hunk 5]
```diff
@@ -81,20 +81,20 @@ scoop update shield
...
+alias shield="docker run -e HOME=/tmp -v $HOME/.config/goto:/tmp/.config/goto --user $(id -u):$(id -g) --rm -it -p 3306:3306/tcp goto/shield:latest"
```
Comment (mabdh) [inline README.md]:
```suggestion
alias shield="docker run -e HOME=/tmp -v $HOME/.config/gotocompany:/tmp/.config/gotocompany --user $(id -u):$(id -g) --rm -it -p 3306:3306/tcp goto/shield:latest"
```

File: buf.gen.yaml [hunk 6]
```diff
@@ -7,9 +7,9 @@ plugins:
   - remote: "buf.build/library/plugins/go-grpc:v1.1.0-2"
     out: "proto"
     opt: "paths=source_relative"
-  - remote: buf.build/odpf/plugins/validate
-    out: "proto"
-    opt: "paths=source_relative,lang=go"
+#  - remote: "buf.build/bufbuild/validate-go"
+#    out: "proto"
+#    opt: "paths=source_relative,lang=go"
```
Comment (mabdh) [inline buf.gen.yaml]:
```suggestion
```

Comment (mabdh) [reply buf.gen.yaml]: just remove it if not needed?

File: cmd/help.go [hunk 7]
```diff
@@ -17,6 +17,6 @@ var authHelp = map[string]string{
 	"long": heredoc.Doc(`
 			Send an additional flag header with "key:value" format.
 			Example:
-				shield create user -f user.yaml -H X-Shield-Email:user@odpf.io
+				shield create user -f user.yaml -H X-Shield-Email:user@goto.io
```
Comment (mabdh) [inline cmd/help.go]:
```suggestion
				shield create user -f user.yaml -H X-Shield-Email:user@gotocompany.com
```

File: cmd/help.go [hunk 8]
```diff
@@ -5,8 +5,8 @@ import "github.com/MakeNowJust/heredoc"
 var envHelp = map[string]string{
 	"short": "List of supported environment variables",
 	"long": heredoc.Doc(`
-			ODPF_CONFIG_DIR: the directory where shield will store configuration files. Default:
-			"$XDG_CONFIG_HOME/odpf" or "$HOME/.config/odpf".
+			goto_CONFIG_DIR: the directory where shield will store configuration files. Default:
+			"$XDG_CONFIG_HOME/goto" or "$HOME/.config/goto".
```
Comment (mabdh) [inline cmd/help.go]:
```suggestion
			GOTOCOMPANY_CONFIG_DIR: the directory where shield will store configuration files. Default:
			"$XDG_CONFIG_HOME/gotocompany" or "$HOME/.config/gotocompany".
```

File: docs/docs/guides/check-permission.md [hunk 9]
```diff
@@ -13,7 +13,7 @@ There are two ways to check a user permission on a resource in shield,
     {`$ curl --location --request POST 'http://localhost:8000/admin/v1beta1/check'
 --header 'Content-Type: application/json'
 --header 'Accept: application/json'
---header 'X-Shield-Email: doe.john@odpf.io'
+--header 'X-Shield-Email: doe.john@goto.io'
```
Comment (mabdh) [inline docs/docs/guides/check-permission.md]:
```suggestion
--header 'X-Shield-Email: doe.john@gotocompany.com'
```

File: docs/docs/guides/managing-group.md [hunk 10]
```diff
@@ -15,7 +15,7 @@ A group in Shield looks like
         "orgId": "4eb3c3b4-962b-4b45-b55b-4c07d3810ca8",
         "metadata": {
             "description": "group for users in data batching domain",
-            "org-name": "odpf"
+            "org-name": "goto"
```
Comment (mabdh) [inline docs/docs/guides/managing-group.md]:
```suggestion
            "org-name": "gotocompany"
```

File: docs/docs/guides/managing-group.md [hunk 11]
```diff
@@ -102,7 +102,7 @@ A group in Shield looks like
     "orgId": "4eb3c3b4-962b-4b45-b55b-4c07d3810ca8",
     "metadata": {
         "description": "group for users in data batching domain",
-        "org-name": "odpf"
+        "org-name": "goto"
```
Comment (mabdh) [inline docs/docs/guides/managing-group.md]:
```suggestion
        "org-name": "gotocompany"
```

File: docs/docs/guides/managing-organization.md [hunk 12]
```diff
@@ -11,8 +11,8 @@ A organization in Shield looks like
     "organizations": [
         {
             "id": "4eb3c3b4-962b-4b45-b55b-4c07d3810ca8",
-            "name": "ODPF",
-            "slug": "odpf",
+            "name": "goto",
+            "slug": "goto",
```
Comment (mabdh) [inline docs/docs/guides/managing-organization.md]:
```suggestion
            "name": "gotocompany",
            "slug": "gotocompany",
```

File: docs/docs/guides/managing-organization.md [hunk 13]
```diff
@@ -34,8 +34,8 @@ A organization in Shield looks like
 --header 'Content-Type: application/json'
 --header 'Accept: application/json'
 --data-raw '{
-  "name": "ODPF",
-  "slug": "odpf",
+  "name": "goto",
+  "slug": "goto",
```
Comment (mabdh) [inline docs/docs/guides/managing-organization.md]:
```suggestion
  "name": "gotocompany",
  "slug": "gotocompany",
```

File: docs/docs/guides/managing-organization.md [hunk 14]
```diff
@@ -96,11 +96,11 @@ A organization in Shield looks like
...
```
Comment (mabdh) [inline docs/docs/guides/managing-organization.md]:
```suggestion
  "name": "gotocompany",
  "slug": "gotocompany",
```

File: docs/docs/guides/managing-organization.md [hunk 15]
```diff
@@ -96,11 +96,11 @@ A organization in Shield looks like
...
   "metadata": {
       "description": "Open DataOps Foundation",
```
Comment (mabdh) [inline docs/docs/guides/managing-organization.md]:
```suggestion
      "description": "Goto company",
```

File: docs/docs/guides/managing-project.md [hunk 16]
```diff
@@ -101,7 +101,7 @@ A project in Shield looks like
   "name": "Project Beta",
   "slug": "project-beta",
   "metadata": {
-      "description": "Project Beta by ODPF"
+      "description": "Project Beta by goto"
```
Comment (mabdh) [inline docs/docs/guides/managing-project.md]:
```suggestion
      "description": "Project Beta by gotocompany"
```

File: docs/docs/guides/managing-relation.md [hunk 17]
```diff
@@ -35,7 +35,7 @@ A relation in Shield looks like
 --data-raw '{
   "objectId": "a9f784cf-0f29-486f-92d0-51300295f7e8",
   "objectNamespace": "entropy/firehose",
-  "subject": "user:doe.john@odpf.io",
+  "subject": "user:doe.john@goto.io",
```
Comment (mabdh) [inline docs/docs/guides/managing-relation.md]:
```suggestion
  "subject": "user:doe.john@gotocompany.com",
```

File: docs/docs/guides/managing-resource.md [hunk 18]
```diff
@@ -62,14 +62,14 @@ There are two ways to create a resource in the shield,
     {`$ curl --location --request POST 'http://localhost:8000/admin/v1beta1/resources'
 --header 'Content-Type: application/json'
 --header 'Accept: application/json'
---header 'X-Shield-Email: admin@odpf.io'
+--header 'X-Shield-Email: admin@goto.io'
```
Comment (mabdh) [inline docs/docs/guides/managing-resource.md]:
```suggestion
--header 'X-Shield-Email: admin@gotocompany.com'
```

File: docs/docs/guides/managing-resource.md [hunk 19]
```diff
@@ -62,14 +62,14 @@ There are two ways to create a resource in the shield,
...
 --data-raw '{
   "name": "test-resource-beta",
   "projectId": "1b89026b-6713-4327-9d7e-ed03345da288",
   "namespaceId": "entropy/firehose",
   "relations": [
     {
-      "subject": "user:john.doe@odpf.io",
+      "subject": "user:john.doe@goto.io",
```
Comment (mabdh) [inline docs/docs/guides/managing-resource.md]:
```suggestion
      "subject": "user:john.doe@gotocompany.com",
```

File: docs/docs/installation.md [hunk 20]
```diff
@@ -42,7 +42,7 @@ $ docker run -p 8080:8080 \
   -e SHIELD_SPICEDB_HOST=spicedb.localhost:50051 \
   -e SHIELD_SPICEDB_PRE_SHARED_KEY=randomkey
   -v .config:.config
-  odpf/shield serve
+  goto/shield serve
```
Comment (mabdh) [inline docs/docs/installation.md]:
```suggestion
  gotocompany/shield serve
```

File: docs/docs/installation.md [hunk 21]
```diff
@@ -33,7 +33,7 @@ Shield provides Docker image as part of the release. Make sure you have Spicedb
 
 ```sh
 # Download docker image from docker hub
-$ docker pull odpf/shield
+$ docker pull goto/shield
```
Comment (mabdh) [inline docs/docs/installation.md]:
```suggestion
$ docker pull gotocompany/shield
```

**Review by ishanarya0 — APPROVED** [review]

**Pull request merged**